pkgs/applications/version-management/srcml/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, cmake, libxml2, libxslt, boost, libarchive, python, antlr }:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  pname = "srcml";
+  version = "0.9.5";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "http://www.sdml.cs.kent.edu/lmcrs/srcML-src.tar.gz";
```

Comment:
This url does not look stable (newer releases will replace the archive), because it does not contain a version number. Is there a different source we can rely on?